ui/src/notifications/Notifications.tsx

```diff
@@ -98,11 +102,10 @@ export default function Notifications({
     <button
       disabled={isMarkReadPending || !hasUnreads}
       className={cn(
-        'whitespace-nowrap text-sm',
+        'small-button whitespace-nowrap text-sm',
         isMobile ? 'small-button' : 'button',
```

Comment:
if you want `small-button` all the time should it just be this?

```suggestion
        'small-button whitespace-nowrap text-sm',
```